### ISSUE TITLE
scandir.stat_result repr 'unnamed field's on Windows

### ISSUE THREAD

**Issue by xflr6** [benhoyt/scandir#47]
```python
>>> import os, scandir
>>> scandir
<module 'scandir' from 'C:\Python27\lib\site-packages\scandir.pyc'>
>>> d = next(scandir.scandir('C:/Python27'))
>>> os.stat(d.path)
nt.stat_result(st_mode=33206, st_ino=0L, st_dev=0, st_nlink=0, st_uid=0, st_gid=0, st_size=2169L, st_atime=1411551593L, st_mtime=1411551594L, st_ctime=1411551593L)
>>> d.stat()
scandir.stat_result(st_mode=33206L, st_ino=0L, st_dev=0L, st_nlink=0L, st_uid=0L, st_gid=0L, st_size=2169L, unnamed field=1411551593L, unnamed field=1411551594L, unnamed field=1411551593L)
```
The fields (e.g. `st_mtime`) still are accessible under their name, so I guess this only concerns the `repr`.

**Comment by benhoyt** [comment]
Interesting, and thanks for the report! I'll look into it.

**Comment by benhoyt** [comment]
Fixed in the commit above, and I've pushed a new version with this fix (1.1) to PyPI: https://pypi.python.org/pypi/scandir
